eng/ignore-links.txt

```diff
@@ -0,0 +1,2 @@
+#PlaceHolder for template links
```

Comment:
I would remove "not-valid-link" and just leave the comment `# Placeholder file for links that need exceptions from link checking`